src/main/java/games/strategy/engine/framework/ProxyRegistries.java

```diff
@@ -0,0 +1,20 @@
+package games.strategy.engine.framework;
+
+import games.strategy.internal.persistence.serializable.PropertyBagMementoProxy;
+import games.strategy.internal.persistence.serializable.VersionProxy;
+import games.strategy.persistence.serializable.ProxyRegistry;
+
+final class ProxyRegistries {
```

Comment:
Think of this class as serving a similar purpose as `java.util.concurrent.Executors` or `java.util.stream.Collectors`, although it currently only hosts pre-created instances rather providing factory methods.  Maybe it's more akin to `g.s.triplea.delegate.Matches`, which provides both?

Regardless, I expect this class to be short-lived and to most likely not make it past the completion of this feature. :smile: